angular/core/components/mute_explanation_modal/mute_explanation_modal.scss

```diff
@@ -1,3 +1,7 @@
+.mute-explanation-modal {
+  max-width: 500px;
```

Comment:
I think we want to try to stick to our existing variables as much as possible. What about $tiny-max-px, which is 480px?